tests/test_run_config_measurement.py

```diff
@@ -252,6 +252,71 @@ def test_is_passing_constraints(self):
         }])
         self.assertFalse(self.rcm5.is_passing_constraints())
 
+    def test_compare_constraints_none(self):
+        """
+        Checks case where either self or other is passing constraints
```

Comment:
Does it "pass" constraints if there is no constraint?

Reply:
Yes